basic/src/test/java/org/jboss/hal/testsuite/test/configuration/webservices/configuration/WebServicesClientConfigurationTestCase.java

```diff
@@ -0,0 +1,225 @@
+            page.navigate();
+            page.switchToClientConfiguration();
+            page.getResourceManager().selectByName(clientConfigurationName);
+            page.switchToProperties();
+            page.getConfig()
+                    .getResourceManager()
+                    .addResource(WebServicesConfigurationPropertyWizard.class)
+                    .key(propertyKey)
+                    .value(propertyValue)
+                    .saveAndDismissReloadRequiredWindowWithState()
+                    .assertWindowClosed();
+            Assert.assertNotNull("Error regarding duplicate property key should be shown", page.getAlertArea());
+        } finally {
+            operations.removeIfExists(createdAddress);
+            administration.reloadIfRequired();
+        }
+    }
+
+    @Test
+    public void addPropertyWithoutKeyShowsError() throws Exception {
+        final String clientConfigurationName = "client_config_" + RandomStringUtils.randomAlphanumeric(7);
+        final Address createdAddress = WEBSERVICES_ADDRESS.and(CLIENT_CONFIGURATION, clientConfigurationName);
+        try {
+            operations.add(createdAddress);
+            administration.reloadIfRequired();
+            page.navigate();
+            page.switchToClientConfiguration();
+            page.getResourceManager().selectByName(clientConfigurationName);
+            page.switchToProperties();
+            page.getConfig()
+                    .getResourceManager()
+                    .addResource(WebServicesConfigurationPropertyWizard.class)
+                    .save();
+            Assert.assertTrue("Validation error regarding empty property name should be visible", page.getWindowFragment().isErrorShownInForm());
+
+        } finally {
+            operations.removeIfExists(createdAddress);
+            administration.reloadIfRequired();
+        }
+    }
+
+    @Test
+    public void removeEndpointConfiguration() throws Exception {
+        final int endpointConfigurationsSize = operations.readChildrenNames(WEBSERVICES_ADDRESS, CLIENT_CONFIGURATION).listValue().size();
+        final String endpointConfigurationName = "my_client_configuration_" + RandomStringUtils.randomAlphanumeric(7);
+        final Address createdAddress = WEBSERVICES_ADDRESS.and(CLIENT_CONFIGURATION, endpointConfigurationName);
+        try {
+            operations.add(createdAddress);
+            administration.reloadIfRequired();
+            page.navigate();
+            page.switchToClientConfiguration();
+            page.getResourceManager()
+                    .removeResource(endpointConfigurationName)
+                    .confirmAndDismissReloadRequiredMessage()
+                    .assertClosed();
+            administration.reloadIfRequired();
+            new ResourceVerifier(createdAddress, client).verifyDoesNotExist();
+            Assert.assertEquals(page.getResourceManager().listResources().size(), endpointConfigurationsSize);
```

Comment:
Please test presence in GUI using `Assert.assertTrue(page.getResourceManager().isResourcePresent(<name>));`